assets/js/editor.js

```diff
@@ -747,7 +755,9 @@ function uploadImage (e, dataTransferAttr, csrf){
         $(e.target).removeClass("selected");
         dragEventNumber = 0;
     }).on("paste", function(e) {
-        if (!e.originalEvent.clipboardData || !e.originalEvent.clipboardData.files.length) {
+        var clipboard = e.originalEvent.clipboardData || [];
+        var data = clipboard.files || clipboard.items || [];
+        if (!clipboard || !data.length) { //    ^^^^^ Edge
```

Comment:
En fait on peut même encore plus simplifier : 
```suggestion
        if (!data.length) { //                  ^^^^^ Edge
```

Reply:
modifié